docs/UserGuide.md

```diff
@@ -78,38 +78,38 @@ We have structured this User Guide in a way so that you can find what you need e
 
    * Other general features
 
-### 3.2 Reading this document <a name="reading-this-document"></a>
+### 2. Reading this document <a name="reading-this-document"></a>
 
 This section introduces you to some key technical terms, symbols and syntax that are used throughout this guide.
 You may want to familarize yourself with them before moving on to the next section.
 
-#### 3.2.1 Terminology related to the GUI
+#### 2.1 Terminology related to the GUI
 
 Figure 2 shows the GUI of **tCheck**, annotated with descriptions for all components of the GUI.
 
 <img src="images/tCheckInfographic.png" width="1000" height="700" />
 Figure 2 - Annotated GUI of tCheck
 
-#### 3.2.2. General Symbols and Syntax 
+#### 2.2. General Symbols and Syntax 
 
 The following table explains the general symbols and syntax used in this User Guide.
+<div style="page-break-after: always;"></div>
 
 | Syntax |  What it means |
 |----------|-------------|
 | `command` |  The grey block specifies an execuable command which can be entered in the command box |
 | _italics_ | Italicised text specifies the terms related to tCheck's GUI |
-|<div markdown="block" class="alert alert-info"> :information_source: </div>  | An exclamation mark indicates that the following text is a note |
-|<div markdown="span" class="alert alert-primary">:bulb:  | An bulb indicates that the following text is a tip |
+|<div markdown="block" class="alert alert-info"> :information_source: </div>  | An exclamation mark indicates that the following text is a note|
 
-#### 3.2.3. Command Syntax and Usage
+#### 2.3. Command Syntax and Usage
 
 The following table explains the technical terms used in this user guide.
 
 | Technical Term | What it means |
 | ---------------| --------------|
-| Prefix | The first word of a command. tCheck uses this command word to decide what kinds of actions it should perform |
-| Command Word | It is the input that follows behind a prefix. It is used to distinguish different kinds of parameters |
-| Parameter | Values given for specific operations |
+| Command Word | The first word of a command. tCheck uses this command word to decide what kinds of actions it should perform. |
+| Prefix | It is the character before every "/". It is used to distinguish different kinds of parameters. |
```

Comment:
instead of "character", perhaps use "character(s)" as some prefixes have more than one character

Reply:
thanks, I am changing it.